package.json

```diff
@@ -25,7 +25,7 @@
     "@storybook/nextjs": "^7.4.6",
     "@storybook/react": "^7.4.6",
     "@storybook/testing-library": "^0.2.2",
-    "@types/node": "latest",
+    "@types/node": "^20.8.6",
```

Comment:
I needed this (`npm i --save-dev @types/node`) to get the `process` typing to work in the `page.tsx` file.

However, now I see that we already had `latest` in there.

Is node typing working for people? If so, this change can probably get reverted/